schema.json

```diff
@@ -4920,7 +4920,7 @@
                   "defaultValue": null
                 },
                 {
-                  "name": "attrs",
+                  "name": "attributes",
```

Comment:
Renaming to be more in line the the rpc endpoint

Reply:
Looks like the title of the PR should include that we updated the GraphQL schema.

Reply:
I would check with other consumers to make sure we don't break them.

Reply:
Updated. I have updated every consumer of this mutation already 👍